bin/build

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+yamlFile="$(ls intersight-openapi-*.yaml)"
+gemVersion="$(grep VERSION lib/intersight_client/version.rb | cut -d "'" -f 2)"
```

Comment:
These 2 changes are notable in that it prevents hardcoding the version numbers for both the yaml file and the gem version, instead preferring pulling them from their respective canonical locations.

Reply:
I think I'd rather have these two as parameters to the script.

`yamlFile` can be problematic if you have more than one version of the openapi yaml in the folder - which can happen when updating to a newer intersight version.

In such case one would also want to increase the gem version, thus getting `gemVersion` from the current version file is not optimal.

Reply:
Can do 👍